core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorAccessControl.java

```diff
@@ -160,18 +160,6 @@ default void checkCanShowCreateTable(ConnectorSecurityContext context, SchemaTab
         denyShowCreateTable(tableName.toString(), null);
     }
 
-    /**
-     * Check if identity is allowed to create the specified table.
-     *
-     * @throws io.trino.spi.security.AccessDeniedException if not allowed
-     * @deprecated use {@link #checkCanCreateTable(ConnectorSecurityContext context, SchemaTableName tableName, Map properties)} instead
-     */
-    @Deprecated
-    default void checkCanCreateTable(ConnectorSecurityContext context, SchemaTableName tableName)
```

Comment:
I suggest we leave the deprecated methods in connector and system access control, and have the new methods pass through to them.  This way we don't break someone who did not realize we changed the signature.

Reply:
I agree this is how this should be implemented in the first place,  in 364 when the new methods were added.
But now, it's not worthwhile to fix it, and we want to remove the deprecated methods at some point in time anyway. IMO the time is now.

Reply:
Wow, didn't realize it was that long ago.  I guess we have what we have